alacritty_terminal/src/ansi.rs

```diff
@@ -983,6 +989,30 @@ where
                 unhandled(params);
             },
 
+            // Hyperlink.
+            b"8" => {
+                if params.len() >= 3 {
```

Comment:
This seems like a good candidate for an early return rather than indenting everything?